cmd/dosa/schema_test.go

```diff
@@ -284,7 +285,7 @@ func TestSchema_Upsert_Happy(t *testing.T) {
 				dl, ok := ctx.Deadline()
 				assert.True(t, ok)
 				assert.True(t, dl.After(time.Now()))
-				assert.Equal(t, 1, len(ed))
+				assert.Equal(t, 4, len(ed))
 				assert.Equal(t, "awesome_test_entity", ed[0].Name)
```

Comment:
Isn't this now a floppy test? There is nothing in the contract that says that awesome_test_entity must be first in the returned list.